TESTING-INSTRUCTIONS.md

```diff
@@ -4,6 +4,13 @@
 
 ## 2.4.0
 
+### Localize string for description #7219
```

Comment:
Can you remove these testing instructions?  We've updated how we add testing instructions and are only adding new "workflows" here now.

Reply:
Done! Thanks for letting me know!